### PR TITLE
9 post a new comment

### PR DESCRIPTION
**Pull request by MrMcSnail**

### CODE REVIEW

**Comment by MrMcSnail** [comment]
Deficiencies in styling for this branch will be resolved on merge with main.

**Comment by dappernerddesigns** [comment]
Your comments are not optimistically rendering. They only show on a refresh. Also, your colour contrast is VERY hard to read. The blue on purple is almost invisible, please consider changing this as I almost couldn't even find your comment form.

Please address the above comments and resubmit your PR

**Comment by dappernerddesigns** [comment]
Your form box also does not support more than one line comments. Consider another approach rather than an input box.

File: src/App.css [hunk 1]
```diff
@@ -228,9 +238,39 @@ button.vote {
...
+
+#submit-btn { 	
+	border: none;
+	font-family: inherit;
+	background-color: var(--colourC);
+	color: white;
+	padding: 5px;
+	font-size: 1.5rem;
+	border-radius: 10px;
+max-width: fit-content;}
```
Comment (elijaharley) [inline src/App.css]: This is looking really nice in terms of contrast now Kyle - good job.

File: src/components/new-comment-form.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+			<textarea
+				type='text'
+				id='comment-body'
+				value={body}
+				onChange={handleChange}
+			/>
+			<input id='submit-btn' type='submit' value='Post Comment' />
+		</form>
+	);
+}
```
Comment (elijaharley) [inline src/components/new-comment-form.jsx:53]: Good to see that you've considered user context here Kyle, and how you want users to be able to interact with your page. Nice to see that you've given feedback to the user if there is an error with their post.

**Review by elijaharley — APPROVED** [review]
Great stuff Kyle - glad to see that you've taken Verity's comments on board and fixed the issues highlighted previously. The comment posted now renders optimistically and persists. It also works on multi-line comments as tested with JeffSum. I noticed that your comments icon is now clickable, but currently goes to an empty page so would suggest taking a look at whether you want this to have a route to just the comments or not to be clickable.

**Pull request merged**